test/unit/utils-specs.js

```diff
@@ -63,35 +63,32 @@ describe('util', function () {
   });
 
   describe('killAllSimulators', function () {
-    it('should call exec thrice if pgrep does not find any running Simulator with Xcode9', async function () {
+    it('should call exec if pgrep does not find any running Simulator with Xcode9', async function () {
       xcodeMock.expects('getVersion').once().withArgs(true).returns(B.resolve(XCODE_VERSION_9));
       execStub.withArgs('xcrun').returns();
-      execStub.withArgs('osascript').returns('');
       execStub.withArgs('pgrep').throws({code: 1});
 
       await killAllSimulators();
-      execStub.calledThrice.should.be.true;
+      execStub.callCount.should.equal(3);
```

Comment:
Should here 2? (I also ran tests on my local)

```
  1) util
       killAllSimulators
         should call exec if pgrep does not find any running Simulator with Xcode9:

      AssertionError: expected 2 to equal 3
      + expected - actual

      -2
      +3

      at Context.<anonymous> (test/unit/utils-specs.js:72:33)
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
      at _next (node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
```

Reply:
fixed